From 3dfddba79dd4c7bb42fb0758cc1199e3fd9c1bd6 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Fri, 21 Sep 2012 12:12:19 +0000
Subject: [PATCH 101/462] Mantis 0024573: Build: extract ReleaseNotes from
 Subversion-Repository <!-- MantisConnect --> added libraries: manisconnect
 svnkit apache-cli added project: releasenotes

git-svn-id: svn://localhost/SelfServiceBuild/trunk@136 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/java/AntProperties.xml | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/SelfServiceBuild/Build/java/AntProperties.xml b/SelfServiceBuild/Build/java/AntProperties.xml
index 5c7c5f4..c7019e5 100644
--- a/SelfServiceBuild/Build/java/AntProperties.xml
+++ b/SelfServiceBuild/Build/java/AntProperties.xml
@@ -142,6 +142,15 @@
 <property name="JarBatikUtil"               value="${DIR_GLOBALS}/ThirdParty/batik/batik-util.jar"/>
 <property name="JarBatikXml"                value="${DIR_GLOBALS}/ThirdParty/batik/batik-xml.jar"/>
 
+<!-- MantisConnect -->
+<property name="JarMCClient"                value="${DIR_GLOBALS}/ThirdParty/mantisconnect/1.1.1.1/mantisconnect-client-api-1.1.1.1.jar"/>
+
+<!-- SVNKit -->
+<property name="JarSVNKit"                  value="${DIR_GLOBALS}/ThirdParty/svnkit/1.7.5/lib/svnkit-1.7.5-v1.jar"/>
+
+<!-- CLI -->
+<property name="JarCLI"                  value="${DIR_GLOBALS}/ThirdParty/commandlineInterface/1.2/commons-cli-1.2.jar"/>
+
 <!-- Sonstiges -->
 <property name="JarCommonsConfiguration"    value="${DIR_GLOBALS}/ThirdParty/jakarta/commons/configuration/1.0/commons-configuration-1.0.jar"/>
 <property name="JarDom4J"                   value="${DIR_GLOBALS}/ThirdParty/dom4j/1.3/dom4j.jar"/>
@@ -178,6 +187,7 @@
 <property name="JarMassaiPrintService"      value="${JarDir}/massai-printservice.jar"/>
 <property name="JarMassaiCLA"               value="${JarDir}/massai-cla.jar"/>
 <property name="JarMassaiCORBASettings"     value="${JarDir}/massai-corba-settings.jar"/>
+<property name="JarMassaiReleaseNotes"      value="${JarDir}/releasenotes.jar"/>
 
 <!-- Definition des IDL-Compilers: OpenORB ======================= -->
 
-- 
2.41.0.windows.1

